pajbot/bot.py

```diff
@@ -60,6 +60,8 @@
 
 SLICE_REGEX = re.compile(r"(-?\d+)?(:?(-?\d+)?)?")
 
+RANDOMCHOICE_ARGUMENT_REGEX = re.compile(r"(\"(\w+)\",? ?)")
```

Comment:
```suggestion 
	RANDOMCHOICE_ARGUMENT_REGEX = re.compile(r"(\"(\"(((|\s)(\w+|)(\s|))+)\",? ?)")
 ```

Reply:
I just tested this and seems to be working fine. Only limitation is non-word characters like periods or commas in a sentence.

Reply:
Resolved in [`a01d4a4` (#1920)](https://github.com/pajbot/pajbot/pull/1920/commits/a01d4a4e5d3c778283252349d400f2f8c91a7b8d)